docs/extend/index.md

```diff
@@ -208,18 +121,15 @@ $ git clone https://github.com/vieux/docker-volume-sshfs
 $ cd docker-volume-sshfs
 $ docker build -t rootfs .
 $ id=$(docker create rootfs true) # id was cd851ce43a403 when the image was created
```

Comment:
Perhaps `rootfs` should be changed to a more descriptive example name (also below where it's passed to `rmi`).